client/my-sites/plugins/minimum-jetpack-version-notice.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { warningNotice } from 'state/notices/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	isJetpackSite,
+	siteHasMinimumJetpackVersion,
+	getSiteAdminUrl,
+	getSiteDomain
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends Component {
+
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		minimumJetpackVersionFailed: PropTypes.bool.isRequired,
+		adminUrl: PropTypes.string,
+		domain: PropTypes.string,
+	};
+
+	showWarning( { minimumJetpackVersionFailed, domain, adminUrl, triggerNotice, translate, siteId } ) {
```

Comment:
Hi @tyxla, the main reason of this component is not directly to avoid adding redux to the plugin components, the main reason is to avoid duplicate code. If notices are not handled in a component, all the line of this file besides imports would be copy pasted to the 3 files that use this component. The notices middleware handles the notice, this components handles the notice creation and handles retrieving the data from redux to check if the notice should be shown and to get the data for the notice creation.